src/components/PizzaToppingCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckSquare, faSquare } from '@fortawesome/free-regular-svg-icons'
+
+function PizzaToppingCard(props) {
+
+  const bootstrapStyle = "m-1 border border-secondary container "
+
+  return (
+
+    <div key={props.name}
+      className={bootstrapStyle}>
+      <div className={"row justify-content-between align-items-center"}>
+        <div className="col-2" >
+              {props.isAdded ?
+                <FontAwesomeIcon icon={faCheckSquare}
+                     onClick={ ()=> props.checkBoxClicked(props.ingredientID)}  />
+              :
+                <FontAwesomeIcon icon={faSquare}
+                     onClick={ ()=> props.checkBoxClicked(props.ingredientID)}  />
+              }
```

Comment:
I am not sure about it, but perhaps it is possible to move the condition into the brackets after icon.
so: <FontAwesomeIcon icon={props.isAdded ? faCheckSquare : faSquare}
                      onClick={ ()=> props.checkBoxClicked(props.ingredientID)}  />
                      
It would be nicer solution.